services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentContentEntity.kt

```diff
@@ -36,7 +37,13 @@ class ApplicationAttachmentContentEntity(
 ) : AttachmentContentEntity(attachmentId, content)
 
 @Repository
-interface HankeAttachmentContentRepository : JpaRepository<HankeAttachmentContentEntity, UUID>
+interface HankeAttachmentContentRepository : JpaRepository<HankeAttachmentContentEntity, UUID> {
+
+    @Query(value = "SELECT * FROM hanke_attachment_content LIMIT 1", nativeQuery = true)
+    fun pickOne(): HankeAttachmentContentEntity?
```

Comment:
This is equivalent to the derived query `findFirstBy()`.
```suggestion
    fun findFirstBy(): HankeAttachmentContentEntity?
```